src/Microsoft.AspNetCore.Razor/Chunks/TagHelperAttributeChunk.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Razor.Chunks
+{
+    public struct TagHelperAttributeTracker
```

Comment:
This is an obscure type in an even more obscure namespace. Who would read the comment?